.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
```

Comment:
Last I checked you had to explicitly ask for container builds with `sudo: false`, which broke various things that time I tried it, so this should continue previous behavior.